DXMainClient/DXGUI/Multiplayer/GameLobby/MultiplayerGameLobby.cs

```diff
@@ -521,7 +522,14 @@ private void RollDiceCommand(string dieType)
 
             BroadcastDiceRoll(dieSides, results);
         }
-
+        private void HandRPSCommand(string ent)
+        {
+            string[] handgame = { "Rock", "Paper", "Scissors" };
+            Random rand = new Random();
+            int index = rand.Next(0, handgame.Length);
+            string reuslts = " selected  " + handgame[index] + "!";
```

Comment:
`reuslts`

Reply:
You shouldn't prepare the string for display when handling the internal logic. Make `HandleHandRPSResult` accepts the index of R/P/S (not a hardcoded English string)